securedrop/manage.py

```diff
@@ -18,8 +19,11 @@
 from db import db_session, init_db, Journalist
 from management.run import run
 
+logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s')
+log = logging.getLogger(__name__)
 
-def reset():  # pragma: no cover
+
+def reset(args):  # pragma: no cover
```

Comment:
Another reason for splitting is that the two PRs that need this change are unrelated and it would not make much sense to have one depending on the other.